m2cgen/interpreters/interpreter.py

```diff
@@ -130,6 +132,23 @@ def interpret_exp_expr(self, expr, **kwargs):
         return self._cg.function_invocation(
             self.exponent_function_name, nested_result)
 
+    def interpret_log_expr(self, expr, **kwargs):
+        if self.logarithm_function_name is NotImplemented:
+            raise NotImplementedError("Logarithm function is not provided")
```

Comment:
Unfortunately, I didn't find anything that we could use as a fallback expression. All methods are based on **reduction** technique  which requires `frexp` implementation.
Refer for example to https://golang.org/src/math/log1p.go